alert/alert.go

```diff
@@ -23,6 +24,11 @@ type Config struct {
 	AWSSecretAccessKey string
 }
 
+func New(config Config, err error) {
+	sentry.CaptureException(err)
+	SendEmail(config, err.Error())
+}
+
 func SendEmail(config Config, body string) {
```

Comment:
There are two places in internal.go that still call `SendEmail`. Could `sendEmail` be private to enforce the use of `New`?

Reply:
I don't think it's worth a breaking change, but it's fine.